apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/welcome-tour/tour-launch.js

```diff
@@ -40,9 +41,9 @@ function LaunchWpcomWelcomeTour() {
 			is_gutenboarding: window.calypsoifyGutenberg?.isGutenboarding,
 			is_manually_opened: isManuallyOpened,
 		} );
-	}, [ isNewPageLayoutModalOpen, isManuallyOpened, show ] );
+	}, [ isNewPageLayoutModalOpen, isManuallyOpened, show, isWelcomeGuideEnabled ] );
 
-	if ( ! show || isNewPageLayoutModalOpen ) {
+	if ( ! show && ! isNewPageLayoutModalOpen ) {
```

Comment:
Do we need to change the condition here?

Reply:
Oh yes, you're right, thanks for noticing! (We don't need to change that) 😄